src/main/webapp/app/markdown-editor/commands/italic.command.ts

```diff
@@ -14,13 +14,15 @@ export class ItalicCommand extends Command {
      */
     execute(): void {
         const selectedText = this.getSelectedText();
+        let textToAdd = '';
 
-        if (selectedText.includes('*')) {
-            const textToAdd = selectedText.slice(1, -1);
+        if (selectedText.charAt(0) === '*' && selectedText.charAt(selectedText.length - 1) === '*') {
```

Comment:
This could be something that is easy to create a method for. `isSelectedTextWithinCommandIdentifiers(text)`.